packages/block-library/src/group/index.js

```diff
@@ -88,6 +90,8 @@ export const settings = {
 		anchor: true,
 		html: false,
 		lightBlockWrapper: true,
+		__experimentalColor: true,
+		customClassName: false,
```

Comment:
This is temporary, I had conflicts between the two hooks "__experimentalColor" and "customClassName". I believe it's a priority issue, customClassName should be the last one to run